src/lib/menu/menu-interface.ts

```diff
@@ -0,0 +1,11 @@
+import {EventEmitter, TemplateRef} from '@angular/core';
+import {MenuPositionX, MenuPositionY} from './menu-positions';
+
+export interface MdMenuInterface {
+  positionX: MenuPositionX;
+  positionY: MenuPositionY;
+  templateRef: TemplateRef<any>;
+  close: EventEmitter<void>;
+  _focusFirstItem: () => void;
+  _setClickCatcher: (bool: boolean) => void;
```

Comment:
If these two are going to be part of this interface, they'll should lose the underscore (which implies "internal api only")